tools/install.py

```diff
@@ -51,21 +51,45 @@ def run(target, local, user, config, toolchain, **kwargs):
 
     # Target location to install to
     if local:
+        target_name = target
         target_binaries_dir = os.path.abspath(os.path.join(target, "binaries"))
         target_toolchain_dir = os.path.abspath(os.path.join(target, "toolchain"))
 
         # Ensure the directories exist
         os.makedirs(target_binaries_dir, exist_ok=True)
         os.makedirs(target_toolchain_dir, exist_ok=True)
     else:
+        target_name = "{0}@{1}".format(user, target)
         target_binaries_dir = "{0}@{1}:/home/{0}/".format(user, target)
         target_toolchain_dir = "{0}@{1}:/usr/".format(user, target)
 
+    # Build directory on the robot
     build_dir = b.binary_dir
 
-    cprint("Installing binaries to " + target_binaries_dir, "blue", attrs=["bold"])
-    files = glob.glob(os.path.join(build_dir, "bin", "*"))
-    subprocess.run(["rsync", "-avPl", "--checksum", "-e ssh"] + files + [target_binaries_dir])
+    # Traverse all binaries, including in subfolders
+    for dirpath, _, filenames in os.walk(os.path.join(build_dir, "bin")):
+        if not "lib" in dirpath:  # lib folder contains libraries, not binaries
```

Comment:
It seems like we used to install libraries - how come this changed?

Reply:
Yea, you kind of need to install the libraries, these are important

Reply:
I definitely got confused somewhere there. But this is now making me wonder why I have to put in all this stuff with making directories on the robot for the subfolders anyway, since I assume `lib` isn't there on initial robot set up? Maybe the error I was getting was actually because of the existing `webots` binary clashing with the new `webots` subfolder. Will test when I'm in the lab.